packages/strapi-provider-email-mailgun/README.md

```diff
@@ -45,7 +45,11 @@ module.exports = ({ env }) => ({
     provider: 'mailgun',
     providerOptions: {
       apiKey: env('MAILGUN_API_KEY'),
-      domain: env('MAILGUN_DOMAIN'), //Required if you have an account with multiple domains
+      // Required if you have an account with multiple domains
+      domain: env('MAILGUN_DOMAIN'),
```

Comment:
Please state this is optional and try to shorten the comment to a single line with the config option. Default should be `api.us.mailgun.net`